### PR TITLE
Update to latest version instead of next version

### PR DESCRIPTION
**Pull request by Senth**
The mod can now update to the latest version and not only to the next version.
Instead of having to run the 'fac update' 10 times if there have been 10 updates to a mod, it is now updated directly to the latest version.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -23,7 +23,7 @@ Or directly from git:
 
 .. code::
 
-    $ pip3 install -e "git+https://github.com/mickael9/fac.git#egg=fac-cli"
+    $ pip3 install -e "git+https://github.com/Senth/fac.git#egg=fac-cli"
```
Comment (Zopieux) [inline README.rst]: I'm not sure we want to do that! :smile:

Comment (Senth) [reply README.rst]: Ahh, I changed that yesterday as I released another project that used fac and wanted those installing fac to have the updated functionality (until the pull request was accepted). Didn't think the commit would be added here as well. Well, I can revert it or create a branch and submit another pull request :) No I know why you should create a branch when creating a pull request ;)

**Comment by Senth** [comment]
Creating a new pull request to fix the problem with the readme and closing this one.

File: README.rst (hunk 1 above)
Comment (Zopieux) [reply README.rst]: No problem, thanks :)

**Pull request closed without merging**